public/app/features/explore/NoDataSourceCallToAction.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { useContext } from 'react';
+import { css } from 'emotion';
+import { ThemeContext, ExtraLargeLinkButton, CallToActionCard } from '@grafana/ui';
+
+export const NoDataSourceCallToAction = () => {
+  const theme = useContext(ThemeContext);
+
+  const message = 'Please add a datasource that supports Explore (e.g., Prometheus)';
```

Comment:
All datasources are supported now, so we can rephrase this:
"Explore requires at least one data source. Once you have added a data source, you can query it here."